apps/wallet/src/ui/app/pages/home/tokens/TokensDetails.tsx

```diff
@@ -200,6 +207,21 @@ function TokenDetails({ coinType }: TokenDetailsProps) {
 	// Avoid perpetual loading state when fetching and retry keeps failing add isFetched check
 	const isFirstTimeLoading = isLoading && !isFetched;
 
+	useEffect(() => {
+		const dismissed = localStorage.getItem('bullshark-interstitial-dismissed');
+		setInterstitialDismissed(Boolean(dismissed));
```

Comment:
`Boolean('false') === true` so probably don't want to do it this way. Probably just check the value directly (dismissed === 'true')